app/utils.py

```diff
@@ -0,0 +1,124 @@
+"""Handle all the user validation"""
```

Comment:
This file will not be required if you use a serializer like marshmallow. Find out how to validate data using marshmallow. Please follow [this link](https://goo.gl/QLxCJe) to see how you can do this.